src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,141 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+
+        Storage.people.clear();
+        user = new User();
+        user.setLogin("Example1");
+        user.setPassword("Password1");
+        user.setAge(30);
+        Storage.people.add(user);
```

Comment:
you only need to have existing user in one test (register existing login), so better move this logic there. You can set up here a valid user (what you already do, just don't add it to a DB) and use it in all the other negative tests. Overall, it's a code duplication